crates/red_knot/tests/cli.rs

```diff
@@ -28,24 +28,22 @@ fn config_override() -> anyhow::Result<()> {
         ),
     ])?;
 
-    case.insta_settings().bind(|| {
```

Comment:
Some more tidy work. I realised I can use a guard to attach the settings automatically

Reply:
Cool!